app/assets/stylesheets/blacklight_gallery/_gallery.scss

```diff
@@ -2,9 +2,8 @@
   .document {
     border-bottom: none;
     display: flex;
-
     min-height: 250px;
-    -webkit-flex: 1 0 250px;
+    flex: 1 0 auto;
```

Comment:
```suggestion
```

I'm not sure we need this at all, if we assume the container uses `.row-cols-*`

![Screen Shot 2022-03-01 at 12 55 39](https://user-images.githubusercontent.com/111218/156247406-2b40aedb-712e-4562-bfc7-4ff8e5a285f5.png)

Reply:
@cbeer the difference I see is that flex-grow (the first value) is 1 here vs 0 with row-cols